mngapi/client.go

```diff
@@ -0,0 +1,237 @@
+package mngapi
+
+import (
+	"bytes"
+	"crypto/rsa"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"path/filepath"
+	"strings"
+	"time"
+
+	jwtgo "github.com/dgrijalva/jwt-go"
+)
+
+const (
+	// RequestTimeout default value to 30 seconds
+	RequestTimeout = time.Duration(30 * time.Second)
+
+	// JWTExpireDuration default value to 1 hour
+	JWTExpireDuration = time.Hour
+
+	//JWTAlgorithm default value to RS256
+	JWTAlgorithm = "RS256"
+)
+
+// HTTPClient interface
+type HTTPClient interface {
+	Do(req *http.Request) (*http.Response, error)
+}
+
+// DefaultClient interface
+type DefaultClient interface {
+	Request(method string, path string, body interface{}) ([]byte, *APIError)
+}
+
+// Client instance
+type Client struct {
+	rootAPIUrl       string
+	endpointPrefix   string
+	jwtIssuer        string
+	jwtSigningMethod jwtgo.SigningMethod
+	jwtPrivateKey    *rsa.PrivateKey
+}
+
+// APIError response from management API
+type APIError struct {
+	StatusCode int      `json:"code"`
+	Error      string   `json:"error,omitempty"`
+	Errors     []string `json:"errors,omitempty"`
+}
+
+var (
+	httpClient HTTPClient
```

Comment:
Please remove this global variable, (same in peatio and barong pkgs)
You can just move it in Client struct.

Reply:
Fixed.